src/test/MsQuicTests.h

```diff
@@ -671,6 +671,13 @@ static const GUID QUIC_TEST_DEVICE_INSTANCE =
 // IOCTL Interface
 //
 
+typedef struct {
+    BOOLEAN UseDuoNic;
+} QUIC_TEST_CONFIGURATION_PARAMS;
+
+#define IOCTL_QUIC_TEST_CONFIGURATION \
+    QUIC_CTL_CODE(0, METHOD_BUFFERED, FILE_WRITE_DATA)
```

Comment:
😄 I like what you did here, slipping this in here at the beginning where there was _just enough_ space.